workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/WorkerGenerationJob.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Improbable.CodeGeneration.FileHandling;
+using Improbable.CodeGeneration.Jobs;
+
+namespace Improbable.Gdk.CodeGenerator
+{
+    public class WorkerGenerationJob : CodegenJob
+    {
+        private readonly List<string> workerTypesToGenerate;
+        private readonly string relativeOutputPath = Path.Combine("improbable", "buildsystem");
+        private readonly string relativeEditorPath = Path.Combine("improbable", "buildsystem", "Editor");
+        private readonly string workerFileName = Path.ChangeExtension("WorkerMenu", ".cs");
+        private readonly string buildSystemFileName = Path.ChangeExtension("Improbable.Gdk.Generated.BuildSystem", ".asmdef");
```

Comment:
Why not just make it a string literal? (and the line above)
Also, won't this replace ".BuildSystem" with ".asmdef"?